### PR TITLE
Remove  PloneFolder, dead code.

### PR DESCRIPTION
**Pull request by jensens**
`PloneFolder` was once used with early Archetypes. This code is dead now.
I did a search over all Plone Github orga and even if it is a bad search term I think there is now evidence of current usage.

### CODE REVIEW

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

**Comment by jensens** [comment]
@jenkins-plone-org please run jobs

File: Products/CMFPlone/PloneTool.py [hunk 1]
```diff
@@ -617,7 +616,7 @@ def browserDefault(self, obj):
         # 7. If nothing else is found, fall back on the object's 'view' action.
         # 8. If this is not found, raise an AttributeError
 
-        # WebDAV in Zope is odd it takes the incoming verb eg: PROPFIND
+        # 0. WebDAV in Zope is odd it takes the incoming verb eg: PROPFIND
```
Comment (thet) [inline Products/CMFPlone/PloneTool.py:619]: should be ``9.``

**Review by thet — APPROVED** [review]
A single typo was all I found.
LGTM and great removal!

**Comment by pbauer** [comment]
L very GTM

File: Products/CMFPlone/PloneTool.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/PloneTool.py:619]: no, the list starts again, it the first action of above list

**Comment by jensens** [comment]
Please merge if you think its fine. I have a next action item depending on this: Removal of `Dublincore.py`.

File: Products/CMFPlone/PloneTool.py (hunk 1 above)
Comment (jensens) [reply Products/CMFPlone/PloneTool.py:619]: Note the above list is the docstring as comment (b/c of security reasons as stated there). The number zero here is a real code comment about the following lines.

**Pull request merged**